src/com/mambu/apisdk/util/RequestExecutorImpl.java

```diff
@@ -139,21 +141,22 @@ public String executeRequest(String urlString, ParamsMap params, Method method,
 		} catch (IOException e) {
 			LOGGER.warning("IOException: message= " + e.getMessage());
 			throw new MambuApiException(e);
+		} finally {
+			httpClient.getConnectionManager().shutdown();
```

Comment:
Doesn't shutting down the connection manager prevent re-using the HTTP connection? Ideally we can re-use the connection for subsequent requests.